### PR TITLE
Allow empty config file

### PR DESCRIPTION
**Pull request by dmitriy-serdyuk**
`yaml` returns `None` when parses an empty string, standard behaviour should be an empty dict in our case.

### CODE REVIEW

**Comment by rizar** [comment]
Could you please make a test for that?

**Comment by dmitriy-serdyuk** [comment]
Done.

**Comment by dmitriy-serdyuk** [comment]
@rizar , is it better?

File: tests/test_config.py [hunk 1]
```diff
@@ -36,6 +46,15 @@ def test_config():
         assert config.data_path == 'manual_path'
         config.new_config = 'new_config'
         assert config.new_config == 'new_config'
-    finally:
+
+
+class TestEmptyConfig(object):
```
Comment (rizar) [inline tests/test_config.py]: But why not merge two tests?

**Comment by dmitriy-serdyuk** [comment]
Oops, forgot about this PR. Fixed.

**Pull request merged**